fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -435,20 +435,16 @@ public TableProperty buildReplicationNum() {
     }
 
     public TableProperty buildPartitionTTL() {
-        if (partitionTTLNumber != INVALID) {
-            return this;
+        if (properties.containsKey(PropertyAnalyzer.PROPERTIES_PARTITION_TTL_NUMBER)) {
+            partitionTTLNumber = Integer.parseInt(properties.get(PropertyAnalyzer.PROPERTIES_PARTITION_TTL_NUMBER));
```

Comment:
will it be allowed to set the ttl number to `-1`?

Reply:
yes, `-1` is diable ttl